code/modules/organs/organ_external.dm

```diff
@@ -875,6 +875,8 @@ Note that amputating the affected organ does in fact remove the infection from t
 
 	switch(disintegrate)
 		if(DROPLIMB_EDGE)
+			if(appearance_flags >= PIXEL_SCALE)
+				appearance_flags -= PIXEL_SCALE
```

Comment:
This is a bitflag, not a number. Don't use mathematical operators on bitflags. If you do it properly, you don't even need the conditional:

```
appearance_flags &= ~PIXEL_SCALE
```

Reply:
Roger that